### PR TITLE
RscriptRequirement: Remove ``flag`` and ``repo``

### PR DESCRIPTION
**Pull request by yash-nisar**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by jayvdb** [comment]
ack 646ef31

File: bears/r/RLintBear.py [hunk 1]
```diff
@@ -24,10 +26,12 @@ class RLintBear:
     LANGUAGES = {'R'}
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
-    REQUIREMENTS = {RscriptRequirement(package='lintr', flag='-e',
-                                       repo='http://cran.rstudio.com'),
-                    DistributionRequirement(apt_get='r-base',
-                                            version='>=3.1.1')}
+    REQUIREMENTS = {
+        AnyOneOfRequirements(
```
Comment (jayvdb) [inline bears/r/RLintBear.py]: These are not logical OR.  `lintr` currently requires R 3.1.1

File: bears/r/FormatRBear.py [hunk 2]
```diff
@@ -22,8 +26,15 @@ class FormatRBear:
...
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
-    REQUIREMENTS = {RscriptRequirement(package='formatR', flag='-e',
-                                       repo='http://cran.rstudio.com')}
+    REQUIREMENTS = {
+        AnyOneOfRequirements(
+            [DistributionRequirement(
+                apt_get='r-cran-formatr',
+                zypper='R-formatR',
+            ),
```
Comment (jayvdb) [inline bears/r/FormatRBear.py]: the indentation here is wrong.

File: bears/r/RLintBear.py (hunk 1 above)
Comment (yash-nisar) [inline bears/r/RLintBear.py]: Dropped the commit.

File: bears/r/FormatRBear.py (hunk 2 above)
Comment (yash-nisar) [inline bears/r/FormatRBear.py]: Fixed.

File: bears/r/FormatRBear.py [hunk 3]
```diff
@@ -22,8 +26,15 @@ class FormatRBear:
...
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
-    REQUIREMENTS = {RscriptRequirement(package='formatR', flag='-e',
-                                       repo='http://cran.rstudio.com')}
+    REQUIREMENTS = {
+        AnyOneOfRequirements(
+            [DistributionRequirement(
+                apt_get='r-cran-formatr',
+                zypper='R-formatR',
+             ),
+             RscriptRequirement(package='formatR'), ]
```
Comment (jayvdb) [inline bears/r/FormatRBear.py]: put the closing bracket on a new line, with its own closing bracket, so these lines dont need to be modified.

File: bears/r/RLintBear.py [hunk 4]
```diff
@@ -24,8 +24,7 @@ class RLintBear:
     LANGUAGES = {'R'}
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
-    REQUIREMENTS = {RscriptRequirement(package='lintr', flag='-e',
-                                       repo='http://cran.rstudio.com'),
+    REQUIREMENTS = {RscriptRequirement(package='lintr'),
```
Comment (jayvdb) [inline bears/r/RLintBear.py]: `package=` is unnecessary.

**Comment by jayvdb** [comment]
ack ef7fd23b dba9066

**Comment by jayvdb** [comment]
reack 535fb60 71c0b95

**Comment by jayvdb** [comment]
@rultor merge

**Pull request merged**